panic: runtime error: slice bounds out of range [8:0]

## Description

I scan a YAML file with the following command:

`./trivy fs -f json   --security-checks vuln,config project-backend/backend-template.yml`

File linked bellow with some information redacted. The file can be used to reproduce the crash. 

## What did you expect to happen?

A successful scan and a JSON output. 

## What happened instead?

```
panic: runtime error: slice bounds out of range [8:0]

goroutine 1 [running]:
github.com/aquasecurity/defsec/parsers/cloudformation/parser.removeLeftMargin({0xc0006b53e0?, 0x5, 0x6})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/parsers/cloudformation/parser/property.go:368 +0xe5
github.com/aquasecurity/defsec/parsers/cloudformation/parser.(*Property).GetJsonBytes(0xc0001cf080, {0x0?, 0x0, 0x1?})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/parsers/cloudformation/parser/property.go:342 +0xfa
github.com/aquasecurity/defsec/parsers/cloudformation/parser.(*Property).GetJsonBytesAsString(...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/parsers/cloudformation/parser/property.go:358
github.com/aquasecurity/defsec/adapters/cloudformation/aws/ecs.getContainerDefinitions(0xc00017b130)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/adapters/cloudformation/aws/ecs/task_definition.go:32 +0x7e
github.com/aquasecurity/defsec/adapters/cloudformation/aws/ecs.getTaskDefinitions({{0xc000194180, 0x54}, {0xc0006b5320, 0x12, 0x12}, {0x1cf39b8, 0x4}, 0xc00107ed20, 0xc00107edb0, 0x0, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/adapters/cloudformation/aws/ecs/task_definition.go:17 +0x2b4
github.com/aquasecurity/defsec/adapters/cloudformation/aws/ecs.Adapt({{0xc000194180, 0x54}, {0xc0006b5320, 0x12, 0x12}, {0x1cf39b8, 0x4}, 0xc00107ed20, 0xc00107edb0, 0x0, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/adapters/cloudformation/aws/ecs/ecs.go:12 +0xb8
github.com/aquasecurity/defsec/adapters/cloudformation/aws.Adapt({{0xc000194180, 0x54}, {0xc0006b5320, 0x12, 0x12}, {0x1cf39b8, 0x4}, 0xc00107ed20, 0xc00107edb0, 0x0, ...})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/adapters/cloudformation/aws/adapt.go:58 +0xa25
github.com/aquasecurity/defsec/adapters/cloudformation.Adapt(...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/adapters/cloudformation/adapt.go:12
github.com/aquasecurity/defsec/scanners/cloudformation/scanner.(*Scanner).Scan(0xc001518500, {0x2208f68, 0xc0010702a0})
        /home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.28.3/scanners/cloudformation/scanner/scanner.go:108 +0x278
github.com/aquasecurity/fanal/config/scanner.Scanner.scanCloudFormation({{0x7ffe57c25120, 0x54}, {0xc0007bd780, 0x2, 0x2}, 0xc000f5b7c0, 0xc00154e3f0, 0xc001518500}, {0x2208f68, 0xc0010702a0}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220404155252-996e81f58b02/config/scanner/scanner.go:151 +0x2a6
github.com/aquasecurity/fanal/config/scanner.Scanner.ScanConfigs({{0x7ffe57c25120, 0x54}, {0xc0007bd780, 0x2, 0x2}, 0xc000f5b7c0, 0xc00154e3f0, 0xc001518500}, {0x2208f68, 0xc0010702a0}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220404155252-996e81f58b02/config/scanner/scanner.go:97 +0x47b
github.com/aquasecurity/fanal/artifact/local.Artifact.Inspect({{0x7ffe57c25120, 0x54}, {0x7f86fca7f6f0, 0xc0008cd300}, {{{0x0, 0x0, 0x0}, {0xc000f5b800, 0x3, 0x4}}}, ...}, ...)
        /home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220404155252-996e81f58b02/artifact/local/fs.go:115 +0x345
github.com/aquasecurity/trivy/pkg/scanner.Scanner.ScanArtifact({{_, _}, {_, _}}, {_, _}, {{0xc0007bd600, 0x2, 0x2}, {0xc0007bd640, ...}, ...})
        /home/runner/work/trivy/trivy/pkg/scanner/scan.go:109 +0x103
github.com/aquasecurity/trivy/pkg/commands/artifact.scan({_, _}, {{0xc0008c1340, 0xc000e00098, {0x21f2a9c, 0x6}, 0x0, 0x0, {0xc0000de000, 0x15}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:257 +0xafe
github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout({_, _}, {{0xc0008c1340, 0xc000e00098, {0x21f2a9c, 0x6}, 0x0, 0x0, {0xc0000de000, 0x15}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:92 +0x465
github.com/aquasecurity/trivy/pkg/commands/artifact.Run({_, _}, {{0xc0008c1340, 0xc000e00098, {0x21f2a9c, 0x6}, 0x0, 0x0, {0xc0000de000, 0x15}}, ...}, ...)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:60 +0xc8
github.com/aquasecurity/trivy/pkg/commands/artifact.FilesystemRun(0xc0008c1340)
        /home/runner/work/trivy/trivy/pkg/commands/artifact/fs.go:49 +0x1c6
github.com/urfave/cli/v2.(*Command).Run(0xc0014e4fc0, 0xc0008c0700)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/command.go:163 +0x5bb
github.com/urfave/cli/v2.(*App).RunContext(0xc000e65040, {0x2208f30?, 0xc0001be020}, {0xc0001ae000, 0x7, 0x7})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/app.go:313 +0xb48
github.com/urfave/cli/v2.(*App).Run(...)
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/app.go:224
main.main()
        /home/runner/work/trivy/trivy/cmd/trivy/main.go:16 +0x4f
```


## Output of run with `-debug`:

```
Incorrect Usage: flag provided but not defined: -debug
```

## Output of `trivy -v`:

```
Version: 0.25.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-04-06 00:10:00.376085335 +0000 UTC
  NextUpdate: 2022-04-06 06:10:00.376084735 +0000 UTC
  DownloadedAt: 2022-04-06 00:30:55.00035862 +0000 UTC
```

## Additional details (base image name, container registry info...):

`backend-template.yml` :
```
Parameters:
  KeyName:
    Type: String
    Default: somename

Resources:
  PrometheusTaskDefinition:
    Type: AWS::ECS::TaskDefinition
    Properties:
      Volumes:
        - Name: project-ecs-stuff
      ContainerDefinitions:
        - Name: project
          Image: 055378293981.dkr.ecr.us-east-1.amazonaws.com/project

        - Name: project-ecs-stuff
          Image: 055378293981.dkr.ecr.us-east-1.amazonaws.com/project-ecs-stuff
 
```

--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/fanal v0.0.0-20220414083417-61bfa9f92483
+	github.com/aquasecurity/fanal v0.0.0-20220416191337-4f5b0beebbd3
 	github.com/aquasecurity/go-dep-parser v0.0.0-20220412145205-d0501f906d90
 	github.com/aquasecurity/go-gem-version v0.0.0-20201115065557-8eed6fe000ce
 	github.com/aquasecurity/go-npm-version v0.0.0-20201110091526-0b796d180798
@@ -75,7 +75,7 @@ require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
-	github.com/aquasecurity/defsec v0.28.4 // indirect
+	github.com/aquasecurity/defsec v0.28.5-0.20220416075528-0f0c8fdf63b8 // indirect
 	github.com/aquasecurity/tfsec v1.8.0 // indirect
 	github.com/aws/aws-sdk-go v1.43.31 // indirect
 	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d // indirect

